From 7efbd3e48a4d158ba06d1711c5cac9e57ed466e4 Mon Sep 17 00:00:00 2001
From: Raffaele Aquilone <raquiloX>
Date: Fri, 11 Jan 2013 17:46:55 +0100
Subject: [PORT FROM MAIN][BT] Broadcom chip SCO PCM configuration

BZ: 79880

Changing the clock rate in the SCO PCM configuration of the
Broadcom BT chip to the lowest one available (0 = 128k), it was
tested and proved to be working

BT chip needs also to be master on PCM SCO link with audio codec

Change-Id: Ic1401dc867700237252fc192d2c9ef9b686e87a2
Orig-Change-Id: I9999776d44532b690355db850f91a213dc4ef99a
Category: device enablement
Domain: CWS.BT-BCM
Origin: internal
Upstream-Candidate: no, not-aosp
Signed-off-by: Raffaele Aquilone <raquiloX>
Reviewed-by: Bellouard, Jean-marie <jean-marie.bellouard@intel.com>
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
Reviewed-on: http://android.intel.com:8080/123801
Reviewed-by: Poignet, Cedric <cedric.poignet@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
---
 include/bt_vendor_brcm.h |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/bt_vendor_brcm.h b/include/bt_vendor_brcm.h
index 8f336bd..2a57952 100644
--- a/include/bt_vendor_brcm.h
+++ b/include/bt_vendor_brcm.h
@@ -283,7 +283,7 @@
     4 : 2048K
 */
 #ifndef SCO_PCM_IF_CLOCK_RATE
-#define SCO_PCM_IF_CLOCK_RATE           4
+#define SCO_PCM_IF_CLOCK_RATE           0
 #endif
 
 /* SCO_PCM_IF_FRAME_TYPE - 0=Short, 1=Long */
@@ -293,12 +293,12 @@
 
 /* SCO_PCM_IF_SYNC_MODE - 0=Slave, 1=Master */
 #ifndef SCO_PCM_IF_SYNC_MODE
-#define SCO_PCM_IF_SYNC_MODE            0
+#define SCO_PCM_IF_SYNC_MODE            1
 #endif
 
 /* SCO_PCM_IF_CLOCK_MODE - 0=Slave, 1=Master */
 #ifndef SCO_PCM_IF_CLOCK_MODE
-#define SCO_PCM_IF_CLOCK_MODE           0
+#define SCO_PCM_IF_CLOCK_MODE           1
 #endif
 
 #define PCM_DATA_FORMAT_PARAM_SIZE      5
-- 
1.7.4.1

